containers/itunes/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Input } from "antd";
+import { debounce } from "lodash-es";
+import { useState, useEffect } from "react";
+import { SongItem, SongResponse, useFetchSongQuery } from "@features/itunes/api/getSongs";
+import { If } from "@common";
+
+export const ITunes = () => {
+  const { Search } = Input;
+  const [songName, setSongName] = useState("");
+  const [display, setDisplay] = useState<SongItem[]>([]);
+  const { data, error, isLoading, isFetching } = useFetchSongQuery("Better");
```

Comment:
we should use the data variable from the hook instead of a separate state variable.